L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -144,6 +144,12 @@
 _phase2_siml1emulator.add( l1tTkMuonsGmt )
 _phase2_siml1emulator.add( l1tSAMuonsGmt )
 
+## fix for low-pt muons, this collection is a copy of the l1tTkMuonsGmt collection 
+## in which we only keep those low pt muons with an SA muon associated to it. Threshold 
+## for this cutoff is configurable. 
+l1tTkMuonsGmtLowPtFix = l1tGMTFilteredMuons.clone()
+_phase2_siml1emulator.add( l1tTkMuonsGmtLowPtFix )
+
```

Comment:
The default applyLowPtFilter is set to False. Should enable it here, otherwise it does nothing.